Packages/vcs/Lib/animate_helper.py

```diff
@@ -579,13 +592,14 @@ def stop(self):
     self.running = False
 
 # Adapted from http://stackoverflow.com/questions/323972/is-there-any-way-to-kill-a-thread-in-python
-class StoppableThread(threading.Thread):
+class StoppableThread(threading.Thread,object):
   def __init__(self):
     threading.Thread.__init__(self)
     self._stop = threading.Event()
     self._running = threading.Event()
     self._running.set()
-
+  def __del__(self):
```

Comment:
Shouldn't be committed. Try using 'git gui' to prepare patches -- it lets you selectively add portions of the working tree's changes so you can commit real changes and leave debugging statements, etc out of the patch.